backend/src/models/db.js

```diff
@@ -1,125 +1,229 @@
 const sqlite3 = require('sqlite3')
-const db = new sqlite3.Database('./db/questions.db')
-
-// NOTE - THIS JUST CREATES A FAKE DATABSE FOR TESTING PURPOSES
-// TODO - REFACTOR
-
-/* Used a tutorial to help get started...
- * This will be changed in the very near future.
- */
-db.serialize(() => {
-    // Create the courses table
-    db.run(`CREATE TABLE courses (
-        cid INTEGER,
-        code TEXT,
-        name TEXT,
-        faculty TEXT,
-        rating INTEGER,
-        uni INTEGER
+
+const databaseName = process.env === 'test' ? ':memory:' : ':memory:'
```

Comment:
We can't do this in production as if the server crashes we lose our entire database. We can do this in dev. Maybe better to add a switch statement.

Reply:
Currently obviously not a problem.